ReactWindows/ReactNative.Shared/Bridge/JavaScriptModuleBase.Generated.cs

```diff
@@ -124,6 +124,5 @@ protected void Invoke(object arg0, object arg1, object arg2, object arg3, [Calle
         {
             Invoke(new[] { arg0, arg1, arg2, arg3 }, caller);
         }
```

Comment:
Going to mark a few of these things for clean up in a separate PR. This is one of them.